### PR TITLE
Setup Angular frontend

### PR DESCRIPTION
**Pull request by ddohler**
This sets up a second Angular app to provide the "real" DRIVER frontend (as compared with the schema editor which is intended to be a super-admin-only tool). @CloudNiner did most of the real work.

### CODE REVIEW

File: web/Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+
+    ngtemplates: {
+      dist: {
+        options: {
+          module: 'webApp',
+          htmlmin: '<%= htmlmin.dist.options %>',
+          usemin: 'scripts/scripts.js'
+        },
+        cwd: '<%= yeoman.app %>',
+        src: 'views/{,*/}*.html',
```
Comment (flibbertigibbet) [inline web/Gruntfile.js]: Might want to make this less limited, to avoid infinite digest cycles like we saw in MUB.

Comment (CloudNiner) [inline web/Gruntfile.js]: +1

Comment (CloudNiner) [inline web/Gruntfile.js]: also change any reference to `views/{,*/}*.html` to `scripts/**/*.html`

File: web/Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+          '<%= yeoman.dist %>/images',
+          '<%= yeoman.dist %>/styles'
+        ],
+        patterns: {
+          js: [[/(images\/[^''""]*\.(png|jpg|jpeg|gif|webp|svg))/g, 'Replacing references to images']]
+        }
+      }
+    },
+
+    // The following *-min tasks will produce minified files in the dist folder
```
Comment (CloudNiner) [inline web/Gruntfile.js]: This commented out block can be deleted

**Comment by kshepard** [comment]
+1, tested, looks good.

**Pull request merged**